source/19-Psalms.usfm.db

```diff
@@ -2956,7 +2956,7 @@
 \q2
 \v 14 That such is God,
 \q our God he it is who shall guide us
-\q2 for ever and ever.
+\q2 [us:forever|cth:for ever] and ever.
```

Comment:
I think the "for ever" here should remain split given the fact that it is part of the phrase "for ever and ever".